doc/devel/documenting_mpl.rst

```diff
@@ -19,6 +24,31 @@ The requirements are as follows (https://github.com/matplotlib/matplotlib/blob/m
 6. pillow
 7. graphviz
 
+.. note::
+
+  * You'll need a minimal working LaTeX distribution for many examples to run.
+  * `Graphviz <http://www.graphviz.org/Download.php>`_ is not a python package, and needs
+    to be installed separately.
+
+General structure
+-----------------
+
+All documentation is built from the :file:`doc/` directory. This directory contains both
+``.rst`` files that contain pages in the documentation, directories that contain more
+``.rst`` files, and configuration files for Sphinx.
+
+.. note::
+
+  An exception to this are the directories :file:`gallery` and :file:`tutorials`, which
+  exist in the root directory. These contain Python files that are built by ``sphinx-gallery``.
```

Comment:
s-g should be be a link instead of double-quoted, similarly to `Sphinx_` (note the underscore) at the top.  The link targets are listed a bit in the middle of nowhere (just before the Figure section), may be worth moving them to the top or the bottom or the file (or just use inline link format like for graphviz above).